From 9cb8854f22898f23de4ef96c2adff42e0f18fbfa Mon Sep 17 00:00:00 2001
From: CI Assistant <yossi.ilkanaev@intel.com>
Date: Fri, 6 Jul 2018 13:54:53 +0200
Subject: [PATCH] Merge pull request #58 in SW_UGW/linux from
 bugfix/DRVLIB_SW-822-add-intel-prefix-in-the-private-dts-for-ethernet-driver
 to master

* commit 'f9f3f98117bcdbda4646ec56ce49bc646c38aad6':
  DRVLIB_SW-778: fix the error report by checkpatch tool in ethernet driver
  DRVLIB_SW-822: add intel prefix in the private dts for ethernet driver
---
 drivers/net/ethernet/lantiq/ltq_eth_drv_xrx500.c | 16 ++++++++--------
 1 file changed, 8 insertions(+), 8 deletions(-)

diff --git a/drivers/net/ethernet/lantiq/ltq_eth_drv_xrx500.c b/drivers/net/ethernet/lantiq/ltq_eth_drv_xrx500.c
index 1e027afbdaca..a38c694dc4b5 100644
--- a/drivers/net/ethernet/lantiq/ltq_eth_drv_xrx500.c
+++ b/drivers/net/ethernet/lantiq/ltq_eth_drv_xrx500.c
@@ -210,7 +210,7 @@ static int get_tsinfo(struct net_device *dev, struct ethtool_ts_info *ts_info)
 	priv = netdev_priv(dev);
 
 	ops = gsw_get_mac_ops(0, (priv->dp_port_id - 2));
-	if(ops)
+	if (ops)
 		ops->mac_get_ts_info(ops, ts_info);
 
 	return 0;
@@ -231,9 +231,9 @@ static const struct ethtool_ops ethtool_ops = {
 	.set_settings		= set_settings,
 	.nway_reset		= nway_reset,
 	.get_link		= ethtool_op_get_link,
-#if IS_ENABLED(CONFIG_LTQ_DATAPATH_PTP1588)	
-	.get_ts_info 		= get_tsinfo,
-#endif	
+#if IS_ENABLED(CONFIG_LTQ_DATAPATH_PTP1588)
+	.get_ts_info		= get_tsinfo,
+#endif
 };
 
 /* open the network device interface*/
@@ -1235,18 +1235,18 @@ static int xrx500_of_iface(struct xrx500_hw *hw, struct device_node *iface,
 	else
 		snprintf(name, sizeof(name), "eth0_%d", hw->num_devs);
 
-	ret = of_property_read_u32(iface, "dp-dev-port",
+	ret = of_property_read_u32(iface, "intel,dp-dev-port",
 				   &dp_dev_port_param);
 	if (ret < 0) {
-		pr_info("ERROR : Property dp-dev-port could not be read from DT for if %s\n",
+		pr_info("ERROR : Property intel,dp-dev-port not read from DT for if %s\n",
 			name);
 		return ret;
 	}
 
-	ret = of_property_read_u32(iface, "dp-port-id",
+	ret = of_property_read_u32(iface, "intel,dp-port-id",
 				   &dp_port_id_param);
 	if (ret < 0) {
-		pr_info("ERROR : Property dp-port-id could not be read from DT for if %s\n",
+		pr_info("ERROR : Property intel,dp-port-id not read from DT for if %s\n",
 			name);
 		return ret;
 	}
